packages/experimental/TabList/src/__tests__/TabList.test.tsx

```diff
@@ -0,0 +1,83 @@
+// import * as React from 'react';
```

Comment:
Will you go back and un-comment this later?

Reply:
Yes. For now, I just need the package to build.

Reply:
We shouldn't include entirely commented out text files into prs. Remove them and add a simple test that will build with this pr